gulpTasks/definition.js

```diff
@@ -33,7 +33,12 @@ gulp.task('cleanDefs', function() {
       .pipe(replace(/readonly/gm, ''))
       .pipe(replace(/undefined/g, 'any'))
       .pipe(replace(/ Record<.*>/g, ' any'))
-      .pipe(replace(/(enum [a-zA-Z_$]+\s{$)((?:\n^\s*[a-zA-Z_$]+ = "[a-zA-Z_$\s]+",$)*)/gm, clearEnumVariableDeclaration))
+      .pipe(
+        replace(
+          /(enum [a-zA-Z_$]+\s{$)((?:\n^\s*[a-zA-Z_$]+ = "[a-zA-Z_$\s]+",?$|\n^\s*[@{}\[\]\w\/*.,\s]+$)*)/gm,
```

Comment:
:(

Could we add a function in this gulp file maybe called `getEnumRegex`. Then, inside we break the regex into parts stored in appropriately named variables, that are combined together to form the return value?

Reply:
I'll try but I'm not sure I remember how it works :trollface: